src/openloco/windows/PlayerInfoPanel.cpp

```diff
@@ -69,11 +70,11 @@ namespace openloco::ui::PlayerInfoPanel
 
     // 0x00509d08
     static widget_t _widgets[] = {
-        make_widget({ 0, 0 }, { 140, 29 }, widget_type::wt_3, 0),                                             // 0
-        make_widget({ 2, 2 }, { 136, 25 }, widget_type::wt_3, 0),                                             // 1
-        make_widget({ 1, 1 }, { 26, 26 }, widget_type::wt_9, 0),                                              // 2
-        make_widget({ 27, 2 }, { 111, 12 }, widget_type::wt_9, 0, image_ids::none, string_ids::tooltip_698),  // 3
-        make_widget({ 27, 14 }, { 111, 12 }, widget_type::wt_9, 0, image_ids::none, string_ids::tooltip_697), // 4
+        make_widget({ 0, 0 }, { 140, 29 }, widget_type::wt_3, 0),
+        make_widget({ 2, 2 }, { 136, 25 }, widget_type::wt_3, 0),
+        make_widget({ 1, 1 }, { 26, 26 }, widget_type::wt_9, 0),
+        make_widget({ 27, 2 }, { 111, 12 }, widget_type::wt_9, 0, string_ids::null, string_ids::tooltip_company_value),
+        make_widget({ 27, 14 }, { 111, 12 }, widget_type::wt_9, 0, string_ids::null, string_ids::tooltip_performance_index),
```

Comment:
Widget type 9 is an image button, so `image_ids::none` is the appropriate value, not `string_ids::null`.
```suggestion
        make_widget({ 27, 2 }, { 111, 12 }, widget_type::wt_9, 0, image_ids::null, string_ids::tooltip_company_value),
        make_widget({ 27, 14 }, { 111, 12 }, widget_type::wt_9, 0, image_ids::null, string_ids::tooltip_performance_index),
```